bin/verify_stored_parameters

```diff
@@ -35,25 +40,29 @@ def VerifyHashnames(codecs, remove_offending_encoders):
     context = encoder.Context(codec, cache_class=encoder.EncodingDiskCache)
     filenames = context.cache.AllEncoderFilenames()
     for filename in filenames:
```

Comment:
This algorithm is getting kind of gnarly, so consider moving the stuff in the try-except block into a helper function.